tutorials/README.md

```diff
@@ -0,0 +1,26 @@
+# Step by step tutorials
+
+## Setup
+
+For running the examples in the tutorials you can either manually install Sockeye, it's dependencies and additional
+tools used or use the Dockerfile we provide, which comes with all batteries included.
+
+### Either: Dockerfile
+
+Coming soon...
+
+<!--we provide a docker file that already includes all prerequisites for running the tutorials.
+If you're using the provided Dockerfile simply skip the `Setup` sections in the tutorials below.-->
```

Comment:
I would avoid contractions in documentation

Reply:
"sections" -> "section"?

Reply:
changed